packages/relay/src/lib/services/transactionPoolService/RedisPendingTransactionStorage.ts

```diff
@@ -15,17 +16,28 @@ export class RedisPendingTransactionStorage implements PendingTransactionStorage
    */
   private readonly keyPrefix = 'txpool:pending:';
 
+  /**
+   * Key for the global pending transactions index.
+   */
+  private readonly globalIndexKey = 'txpool:pending:txns';
```

Comment:
Should we make this a dependant of keyPrefix?

```suggestion
private readonly globalIndexKey = `${keyPrefix}txns`;
```

Also should we make it more clear? globalIndexKey is quite confusing. Could it be globalPendingTxsKey or something? and the value could be `${keyPrefix}global`?

Reply:
done